solveengine/examples/usage.py

```diff
@@ -8,7 +8,7 @@
 #
 # the token
 token = "2prMEegqCAFB5eqmfuXmGufyJ+PcMzJbZaQcvqrhtx4="
-staticmethod
+
```

Comment:
seems you did not run this as the method names in this files have not be corrected